dle/localdeploy.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+mv .env.example .env
+sed '/MEM_LIMIT.*/d' .env > .env.tmp
+mv .env.tmp .env
+sed '/PROVISION_ES.*/d' .env > .env.tmp
+mv .env.tmp .env
+
+# Uncomment these if database has never been created. May take several hours.
+#echo "LOAD=True" >> .env
+#echo "PROVISION_ES=True" >> .env
+
+echo "MEM_LIMIT=4294967296" >> .env
```

Comment:
This should probably get tweaked based on the EC2 size. What stats are we running? We want 1GB for Kibana regardless, ~1-2GB for the host, and probably the rest for Elastic ... That would mean that we're not going to run other ad-hoc tasks using Django on the EC2 unless we adjust those downward.

Reply:
Whoops this is local sorry. Not sure why this is needed?

Reply:
In the past, I needed 4GB to run locally. Not sure if that's changed. Kept for now.